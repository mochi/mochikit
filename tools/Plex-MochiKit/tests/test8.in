ftang ftang ftangftang ftangfta ftang


