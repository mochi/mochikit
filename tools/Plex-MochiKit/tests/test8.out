test8.in   1   0 one_ftang  'ftang'
test8.in   1   6 one_ftang  'ftang'
test8.in   1  12 two_ftangs 'ftangftang'
test8.in   1  23 one_ftang  'ftang'
test8.in   1  28 one_fta    'fta'
test8.in   1  32 one_ftang  'ftang'
test8.in   4   0 None       ''
